docs/tutorials/intro_tutorial.rst

```diff
@@ -713,10 +713,11 @@ At every step of the model, the datacollector will collect and store the
 model-level current Gini coefficient, as well as each agent’s wealth,
 associating each with the current step.
 
-We run the model just as we did above. Now is when an interactive
-session, especially via a Notebook, comes in handy: the DataCollector
-can export the data it’s collected as a pandas DataFrame, for easy
-interactive analysis.
+We run the model just as we did above. Now is when an interactive session, especially via a Notebook, comes in handy: the DataCollector can export the data its collected as a pandas\* DataFrame, for easy interactive analysis.
+
+\*If you are new to Python, please be aware that pandas is already installed as a dependency of Mesa and that `pandas`_ is a "fast, powerful, flexible and easy to use open source data analysis and manipulation tool". pandas is great resource to help analyze the data collected in your models
```

Comment:
*Pandas.

Reply:
@rht I went back and forth on that in the end I went with the fact that they did not capitalize it on their [docs](https://pandas.pydata.org/) but I am good either way.... capitalize?

Reply:
Ugh, I see. It's so hard to keep track of the software that insist that their proper name is lower cased! Even Wikipedia does it in lowercase: https://en.wikipedia.org/wiki/Pandas_(software). I guess we have to respect the lower casing, just like in macOS, nginx, and so on.